core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/remove.kt

```diff
@@ -10,15 +10,44 @@ import kotlin.reflect.KProperty
 
 // region DataFrame
 
-public fun <T> DataFrame<T>.remove(columns: ColumnsSelector<T, *>): DataFrame<T> = removeImpl(allowMissingColumns = true, columns = columns).df
+// region remove
+
+public fun <T> DataFrame<T>.remove(columns: ColumnsSelector<T, *>): DataFrame<T> =
+    removeImpl(allowMissingColumns = true, columns = columns).df
+
 public fun <T> DataFrame<T>.remove(vararg columns: KProperty<*>): DataFrame<T> = remove { columns.toColumns() }
+
 public fun <T> DataFrame<T>.remove(vararg columns: String): DataFrame<T> = remove { columns.toColumns() }
+
 public fun <T> DataFrame<T>.remove(vararg columns: AnyColumnReference): DataFrame<T> = remove { columns.toColumns() }
-public fun <T> DataFrame<T>.remove(columns: Iterable<AnyColumnReference>): DataFrame<T> = remove { columns.toColumnSet() }
+
+@Deprecated(
+    message = "It will be removed in the 0.10.0 release",
+    level = DeprecationLevel.WARNING,
```

Comment:
Could be level ERROR

Reply:
It doesn't work with the ERROR level, because it depends on the function in implementation with the same level of deprecation and it will be not compiled

Reply:
Oh, do you suggest change the implementation? In this way it should work, I suppose. I started to think about binary compatibility and don't think about replacement of the implementation.